inst/htmljs/animint.js

```diff
@@ -1610,6 +1610,20 @@ var animint = function (to_select, json_file) {
     };
     doActions(enter);  // DO NOT DELETE!
     var has_tooltip = g_info.aes.hasOwnProperty("tooltip");
+    function positionTooltip(tooltip, content) {
+      var mouseX = 0, mouseY = 0;
+      try {
+        mouseX = (d3.event && d3.event.pageX) || 0;
+        mouseY = (d3.event && d3.event.pageY) || 0;
+      } catch (e) {
+        console.error("Error getting mouse position:", e);
+      }
+      tooltip
+        .html(content)
+        .style("left", (mouseX + TOOLTIP_HORIZONTAL_OFFSET) + "px")
+        .style("top", (mouseY - TOOLTIP_VERTICAL_OFFSET) + "px")
+        .style("opacity", 1);
```

Comment:
I see this opacity 1 was only present in one of the two versions below, is that important to keep? or is it ok to do opacity 1 in both now?